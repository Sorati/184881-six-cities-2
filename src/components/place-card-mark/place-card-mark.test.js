import React from 'react';
import renderer from 'react-test-renderer';
import PlaceCardMark from './place-card-mark';

it(`Place card mark correctly renders with mark`, () => {
  const tree = renderer.create(< PlaceCardMark title = {
    `Premium`
  } />).toJSON();
  expect(tree).toMatchSnapshot();
});
