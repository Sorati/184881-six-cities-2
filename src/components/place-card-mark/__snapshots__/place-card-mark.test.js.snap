// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Place card mark correctly renders with mark 1`] = `
<div
  className="place-card__mark"
>
  <span>
    Premium
  </span>
</div>
`;
